doc/sphinx-guides/source/installation/provenance.rst

```diff
@@ -0,0 +1,92 @@
+Provenance
+==========
+
+Provenance is metadata that describes the history of a digital object: where it came from, how it came to be in its present state, who or what acted upon it, etc. See "Data Provenance" in the :doc:`/user/dataset-management.rst` section of the User Guide for more on this feature from the end user perspective.
+
+.. contents:: |toctitle|
+  :local:
+  
+Introduction
+------------
+
+Support for provenance within Dataverse is built on top of the "prov-cpl" project at https://github.com/ProvTools/prov-cpl and "CPL" stands for "Core Provenance Library."
+
+Installation
+------------
+
+Here is an overview of the steps required to install the provenance service:
+
+- install an RPM
+- create, configure and secure a PostgreSQL database
+- install, configure, and secure a REST service
+
+Each of these steps is described in detail below.
+
+Installation of the provenance service is only supported on RHEL/CentOS. CentOS 7 was used when writing these instructions.
+
+Install ``dataverse-provenance`` RPM
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Download this :download:`dataverse-provenance RPM <../_static/installation/files/home/rpmbuild/rpmbuild/RPMS/x86_64/dataverse-provenance-0.1-1.x86_64.rpm>` and install it with:
```

Comment:
guides.dataverse.org isn't currently available over https; distributing rpms over http is something that it's probably worth avoiding.

Reply:
That's a fair point but no one has complained (that I'm aware of) about downloading the rapache RPM over plain HTTP: http://guides.dataverse.org/en/4.8.5/installation/r-rapache-tworavens.html#c-rapache

We did have a very security-minded individual take a look at our dev guide a couple years ago. You might enjoy reading through this comment, @pameyer: https://github.com/IQSS/dataverse/issues/2863#issuecomment-199322114

Reply:
If I installed rapache, it would be an issue I'd raise.